### PR TITLE
[PB-1748] Check notice request existence

### PR DESCRIPTION
**Pull request by JIAZHEN**
90% review

- Add setup to README
- Check xml notice request existence
- Able to send error to self-hosted sentry and hosted sentry.

Related PR
https://github.com/globaldev/labourforce-event-adapters/pull/95

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,31 @@
...
+
+## Install & Setup
+
+Get the code from [GitHub][router]:
+
+    cd ~/Projects
+    git clone git@github.com:globaldev/airbrake-proxy.git
+    cd airbrake-proxy
+
+Installing the dependancies:
```
Comment (chiting) [inline README.md]: typo: `dependencies`

File: README.md [hunk 2]
```diff
@@ -1 +1,31 @@
...
+
+    npm config set strict-ssl false
+
+Create configuration files by copying the example configuration:
+
+    cp config/config.json.example config/config.json
+
+## Running
+
+    cd ~/Projects/wld-api-router
```
Comment (chiting) [inline README.md]: should be `~/Projects/airbrake-proxy`

**Comment by chiting** [comment]
LGTM

**Comment by mendable** [comment]
👍

**Comment by timtait** [comment]
👍

**Comment by ianwinter** [comment]
👍 🍍 💥

**Pull request merged**